packages/docs/src/routes/docs/deployments/aws-lambda/index.mdx

```diff
@@ -38,8 +38,8 @@ npm run build
 
 ## Deploy to AWS
 
-Before deploy is need setup your [AWS Credentials](https://www.serverless.com/framework/docs/providers/aws/guide/credentials) underhood
-the adapter  use [serverless](https://www.serverless.com/).
+Before deployment, you need to set up your [AWS Credentials](https://www.serverless.com/framework/docs/providers/aws/guide/credentials). Under the hood, 
+the adapter uses [serverless](https://www.serverless.com/).
```

Comment:
You can add your name `- WilliamEspegren`in this file too 👍

Reply:
Thanks that you saw it! Will do it right away 🙏